app/styles/_admin.scss

```diff
@@ -2558,7 +2558,8 @@ a:focus {
 .nav-tabs-custom > .nav-tabs > li {
   border-top: 3px solid transparent;
   margin-bottom: -2px;
```

Comment:
Since you're using `%` for the sake of uniformity, can you replace the other `px` values with the `%` too ?